code/game/objects/items/weapons/holy_weapons.dm

```diff
@@ -421,7 +421,10 @@
 /obj/item/nullrod/armblade/mining/pickup(mob/living/user)
 	..()
 	flags += ABSTRACT
-	return FALSE
+
+/obj/item/nullrod/armblade/mining/dropped(mob/living/user)
+	..()
+	flags -= ABSTRACT
```

Comment:
not the proper way to do bit operations, you need to make sure that the ABSTRACT flag bit is switched off and not just merely subtracting the flag from flags. This could actually potentially turn the ABSTRACT flag back on if you're not careful

Reply:
Updated